Create strings.xml

uDEV2019 opened the pull request:
german translation

uDEV2019 committed:
Create strings.xml

german translation

warren-bank commented:
Over the past few days, I wrote [tooling](https://github.com/warren-bank/node-translate-android-strings) to automatically generate string translations to all languages supported by the [IBM Cloud](https://github.com/warren-bank/node-ibm-watson-language-translator/blob/master/.etc/docs/IBM-Cloud-account.md) Language Translator service.

I understand that computer generated translations are never perfect, but my aim isn't production-level quality.. only that it's sufficiently usable. How does [this German translation](https://github.com/warren-bank/Android-Mock-Location/blob/master/android-studio-project/Mock-my-GPS/src/main/res/values-de/strings.xml) look? Is it understandable, or mostly gibberish?

uDEV2019 committed:
Update strings.xml

Updated and removed the not mandatory strings.
Better soultion than IBM Cloud Translation Service.

uDEV2019 committed:
Merge branch 'service' into service

uDEV2019 commented:
The service from IBM isn't that good, i think...

some exmples that are gibberish:
- PreferencesActivity_field_2
- MainActivity_actionbar_item_3
- error_missing_required_value

warren-bank commented:
Sorry for the confusion with the commit history;
I didn't like github's automatic commit message.. so I did a forced push to rewrite it.

In any case, thanks for your help with the translation; I really do appreciate it.
I speak one language (and only just barely).. so it's needed :smiley:
